apps/svelte.dev/content/blog/2025-05-01-whats-new-in-svelte-may-2025.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "What's new in Svelte: May 2025"
+description: 'Svelte Summit soon! Plus, CLI and Adapter Updates'
+author: Dani Sandoval
+authorURL: https://dreamindani.com
+---
+
+Only a week until [Svelte Summit Spring 2025](https://www.sveltesummit.com/)! If you can't make it in-person to Barcelona, you can purchase a virtual ticket [here](https://www.sveltesummit.com/virtual-ticket).
+
+But before that, we've got a few updates to highlight...
+
+## What's new in Svelte and SvelteKit
+
```

Comment:
It'd probably be worth mentioning the new async proposal. Not sure if you want to do that here or above, but that's what the team has been spending most time on recently and why this month is otherwise slimmer than normal 

https://github.com/sveltejs/svelte/pull/15844
https://github.com/sveltejs/svelte/discussions/15845